docs/conan.md

```diff
@@ -0,0 +1,147 @@
+# Using dependencies from Conan
+
+[Conan](https://conan.io/) is a package manager for C/C++. We provide prebuilt binary dependencies for some platforms that are used by our CI, but they can also be consumed by users to build VCMI. However, it's not required to use only the prebuilt binaries: you can build them from source as well.
+
+## Supported platforms
+
+The following platforms are supported and known to work, others might require changes to our [conanfile.py](../conanfile.py) or upstream recipes.
+
+- **macOS**: x86_64 (Intel) - target 10.13 (High Sierra), arm64 (Apple Silicon) - target 11.0 (Big Sur)
+- **iOS**: arm64 - target 12.0
+
+## Getting started
+
+1. [Install Conan](https://docs.conan.io/en/latest/installation.html)
+2. Execute in terminal: `conan profile new default --detect`
+
+## Download dependencies
+
+0. If your platform is not on the list of supported ones or you don't want to use our prebuilt binaries, you can still build dependencies from source or try consuming prebuilt binaries from the central Conan repository - [ConanCenter](https://conan.io/center/). In this case skip to the next section directly.
+
+1. Check if your build environment can use the prebuilt binaries: basically, that your compiler version (or Xcode major version) matches the information below. If you're unsure, just skip to next step.
+
+    - macOS: libraries are built with Apple clang 13 (Xcode 13.4.1), should be consumable by Xcode and Xcode CLT 13.x
```

Comment:
I use Xcode 11, does it mean that I cannot just download necessary reps?

Reply:
Correct. For iOS you can use legacy pre-built deps, for macos - from brew. Or build from source for any platform.
In the future this can change of course.

Reply:
but now there is no source of truth for brew. So If wiki has mistake, it significantly increases entry level for developers

Reply:
Does our CI try to build using brew? Not necessary to publish artifacts, just check?

Reply:
it used brew in the past, but it's not a viable option for long-term: in the beginning of next year GitHub will remove the 10.15 runner which means that only users on macOS 11 and later will be able to use CI's artefact. With Conan we target 10.13 at the moment as a sensible baseline, but I believe dependencies should be buildable to a lower macOS as well.